src/deion/ui/views/GodMode.js

```diff
@@ -126,9 +132,7 @@ class GodMode extends React.Component {
             const numPlayoffTeams = 2 ** numRounds - numPlayoffByes;
             if (numPlayoffTeams > this.props.numTeams) {
                 throw new Error(
-                    `${numRounds} playoff rounds with ${numPlayoffByes} byes means ${numPlayoffTeams} teams make the playoffs, but there are only ${
```

Comment:
I think the content here is the same, my IDE must've messed up formatting

Reply:
No worries - it happens sometimes when there is a new version of prettier that slightly changes the formatting rules, but it doesn't see a file until that file is actually edited.